### ISSUE TITLE
Reproducing 'vaihingen' results

### ISSUE THREAD

**Issue by mikigom** [shirgur/ACDRNet#1]
Thank you for your great work and sharing code.

Could you give me exact commend line to reproduce vaihingen dataset results in your paper?
I have struggled to reproduce the results.

**Comment by Just5D** [comment]
+1 
win10 pytorch1.4 
python train.py --data-path .\dataset\testbuildings --train-dataset buildings

D:\ProgramData\Anaconda3\envs\acdrnetou\lib\site-packages\torch\optim\lr_scheduler.py:122: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
0it [00:02, ?it/s]
0it [00:01, ?it/s]
D:\ProgramData\Anaconda3\envs\acdrnetou\lib\site-packages\numpy\core\fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
D:\ProgramData\Anaconda3\envs\acdrnetou\lib\site-packages\numpy\core\_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
0it [00:01, ?it/s]
0it [00:01, ?it/s]
0it [00:02, ?it/s]
0it [00:01, ?it/s]

**Comment by huaxinxiao** [comment]
+1. Cannot reproduce the results.

**Comment by jasmine789** [comment]
Hi Guys, May I ask that have you solve the problem or what the mIoU did you get? Currently, My IoU for the building dataset is only about 0.5(highest), do you get some similar results as me? Really appreciated it if you can provide some suggestions/feedback.

**Comment by mahfuzmohammad** [comment]
Hi Guys, I also tried to reproduce the result. I got the best mAP of around 0.3. @shirgur Any suggestion? Is it possible to publish the trained model?

**Comment by talshaharabany** [comment]
Hi Guys, 

check this out for buildings segmentation - https://github.com/talshaharabany/DeepACM2D
@mahfuzmohammad @mikigom @Just5D @huaxinxiao @jasmine789
